ods_ci/tests/Resources/Page/Pipelines/OpenShiftPipelines.robot

```diff
@@ -0,0 +1,34 @@
+*** Settings ***
+Documentation       Test suite for OpenShift Pipeline
+Resource            ../../../Resources/RHOSi.resource
+Resource            ../../../Resources/ODS.robot
+Resource            ../../../Resources/Common.robot
+Resource            ../../../Resources/Page/ODH/ODHDashboard/ODHDashboard.robot
+Library             DateTime
+Library             ../../../../libs/DataSciencePipelinesAPI.py
+
+
+*** Variables ***
+${REDHAT_OPENSHIFT_PIPELINES_YAML}=            ods_ci/tests/Resources/Files/redhat-openshift-pipelines.yaml
+
+
+*** Keywords ***
+Data Science Pipelines Suite Setup
+    [Documentation]    Data Science Pipelines Suite Setup
+    Set Library Search Order    SeleniumLibrary
+    RHOSi Setup
+    Install Red Hat OpenShfit Pipelines
+
+Install Red Hat OpenShfit Pipelines
+    [Documentation]    Install Red Hat OpenShfit Pipelines
+    Oc Apply    kind=Subscription    src=${REDHAT_OPENSHIFT_PIPELINES_YAML}     namespace=openshift-operators
+    ${pod_count}    Get Redhat Openshift Pipelines
+    Should Be True    ${pod_count} == 1    All the pods were created
+
+Install DataSciencePipelinesApplication CR
```

Comment:
As written above in another comment, we can consolidate keywords related to Data Science Pipelines in RHODS in ods_ci/tests/Resources/Page/ODH/ODHDashboard/ODHDataSciencePipelines.robot

If the intent of this keyword is to create a DataSciencePipelineApplication, I think it could be renamed to `Create DataSciencePipelinesApplication` and have the yaml as argument

```
Create DataSciencePipelinesApplication
[Arguments]     ${project}  ${yaml}
```